script.js

```diff
@@ -0,0 +1,75 @@
+const inputText = document.querySelector('.input-text')
+const addItem = document.querySelector('.add-item')
+const myList = document.querySelector('.todo__list')
+
+const buy = []
+
+ listItem = () => {
+    const listBuy = document.createElement('li')
+    listBuy.classList.add('todo__item')
+
+    buy.forEach((value) => {
+        listBuy.textContent = value.buy
+        myList.appendChild(listBuy)  
+    })
+}
+
+buttonAddItem = () => {
+    addItem.addEventListener("click", (event) =>{ 
+        if(inputText.value === "") {return alert("Please write any text")}
+        else {buy[buy.length] = {buy : inputText.value}}
```

Comment:
you don't need `else` as in your `if` you have `return`.
Code won't reach this place, so else is redundant

Reply:
```
buy = buy.concat([
     {
           buy: inputText.value
     }
])
```

Reply:
also you need to use `let` to make it work